### ISSUE TITLE
[Feature Request] afterBuild function in quasar.conf to run code after the electron app has been packaged

### ISSUE THREAD

**Issue by ExNG** [quasarframework/quasar-cli#95]
This is a proposed solution for #93 , in which now in v0.15 it is missing the possibility to customize the Electron build script. 

A simple solution would be to add a function to the `electron` field of the `quasar.conf.js` called for example `afterBuild()` to give users the ability to run their own code after the app has been packaged.

Code example:

```
electron: {
  // optional; webpack config Object for
  // the Main Process ONLY (/src-electron/main-process/)
  extendWebpack (cfg) {
    // PLUGINS ARE RUN BEFORE THE ELECTRON BUILD IS COMPLETE
  },

  afterBuild: function (appPaths) {
    // CODE TO RUN AFTER THE ELECTRON BUILD IS FINISHED
    // appPaths would be useful to perform operations on the build package
  }
}
```

**Comment by rstoenescu** [comment]
Hi,

This is essential same as doing:
```bash
$ quasar build -m electron
$ node my_script.js
```

So don't think it's worth adding extra functionality to quasar.conf.

**Comment by rstoenescu** [comment]
I think I get what you want. Reopening ticket.

**Comment by rstoenescu** [comment]
Pushed commit which adds quasar.conf > electron.beforePackaging()
** Available in next release: 0.15.13**

Examples:

```js
electron: {
  beforePackaging ({ appPaths, unpackagedDir }) {
    // do something...
  }
}
```
```js
electron: {
  beforePackaging ({ appPaths, unpackagedDir }) {
    return new Promise((resolve, reject) => {
      .... resolve() ...
    })
  })
}
```

Please note that the Electron build is a three step process. First step is to bundle the renderer process. Second is to bundle the main process. Finally, last step is to use Electron Packager to package the app into an executable.
`beforePackaging` is executed between step 2 and step 3.

Anything after step 3 can easily be executed after Quasar CLI finishes its job.
